### DISCUSSION TITLE
The prediction of energy has an overall deviation, but the training of force is good

### DISCUSSION THREAD

**Discussion by KK33999** [deepmodeling/deepmd-kit#4666] [Q&A] [answered]
Hello everyone, when I was using DeepMD-kit (v.3.0.2) for training, the predictions of force for DFT and DP were in good agreement. However, there is an overall deviation between the predictions of energy (as shown in the figure). In response to this situation, I have the following four questions: 

1. Can the potential function in this case be used for future research? 
2. In the final presentation (such as in a paper), can we artificially give a whole offset?
3. How to avoid this situation from happening? 
4. What are the reasons for this. 
![82369eb6b90c7c5269a6498fd7e656e](https://github.com/user-attachments/assets/0ca1bb66-a3d9-4bf1-a895-1d21adf29794)

Thank you in advance.

**Comment by wanghan-iapcm** [comment, marked as answer]
I suppose you are using the smooth edition of the DP descriptor (`"se_e2_a"`)

We observe a constant shift in the energy prediction. To fix the issue one may try:
1. increase the number of training steps. A production train usually has 4M-16M training steps.
2. train model again with pretrained model parameters, large energy prefactors and smaller start learning rate. you may revise your input script (name it by second_round_input.json) as 
```json
"learning_rate": {
        "type": "exp",
        "start_lr": 0.0001,
        "stop_lr": 3.51e-08,
        "_comment": "that's all"
    },
    "loss": {
        "start_pref_e": 1,
        "limit_pref_e": 1,
        "start_pref_f": 1,
        "limit_pref_f": 1,
        "start_pref_v": 1,
        "limit_pref_v": 1,
        "_comment": " that's all"
    },
```
and execute the command
```sh
dp train --init-model model.ckpt second_round_input.json
```

**Reply by KK33999** [reply]
I use DPA-2 version. I have achieved better energy accuracy by increasing the number of training steps. Thank you again for your detailed and patient answer. So do you think the training results in above situation should be unqualified ？

**Reply by wanghan-iapcm** [reply]
by DPA-2 one may use significantly less training steps for production, usually 1M-4M.
